src/scheduler.rs

```diff
@@ -38,14 +38,6 @@ impl Context {
         self.scheduler.read().await.interrupt_inbox(info).await;
     }
 
```

Comment:
Should this Inbox part stay or deleted as well?

Reply:
in contrast to the two removed functions, this function is still in use.